subresource-loading.bs

```diff
@@ -438,15 +438,55 @@ detailed motivation.
 To <dfn>parse a web bundle string</dfn>, given a [=string=] |sourceText| and a
 [=URL=] |baseURL|:
 
-1. Let |parsed| be the result of [=parse JSON into Infra values|parsing JSON
-   into Infra values=] given |sourceText|.
-1. If |parsed| is not a [=map=], then throw a {{TypeError}} indicating that the
-   top-level value needs to be a JSON object.
-1. Create a [=web bundle parse result=] from |parsed| and |baseURL|, and return
-   the [=web bundle parse result=].
-
-ISSUE: Specify the details. Throw a {{TypeError}} etc. on a parse error.
-Relative URLs are resolved on |baseURL|.
+1.  Let |parsed| be the result of [=parse JSON into Infra values|parsing JSON
+    into Infra values=] given |sourceText|.
+1.  If |parsed| is not a [=map=], then throw a {{TypeError}} indicating that the
+    top-level value needs to be a JSON object.
+1.  If |parsed|["`source`"] does not [=map/exist=], then throw a {{TypeError}}.
+1.  If |parsed|["`source`"] is not a [=string=], then throw a {{TypeError}}.
+1.  Let |source| be the result of [=URL parser|parsing=] |parsed|["`source`"]
+    with |baseURL| as the base URL.
+1.  If |source| is null, then throw a {{TypeError}}.
+1.  Let |credentials| be "`same-origin`".
+1.  If |parsed|["`credentials`"] [=map/exists=], then:
+    1. If |parsed|["`credentials`"] is "`omit`", then set |credentials| to
```

Comment:
Just to confirm, this is case-sensitive comparison (i.e. https://infra.spec.whatwg.org/#string-is), correct?

Reply:
Yeah, case-sensitive comparison is intentional. Thanks for the link for "string-is". I didn't know "string-is" is defined.